tlslite/constants.py

```diff
@@ -895,14 +906,29 @@ class CipherSuite:
     tls12Suites.extend(sha384Suites)
     tls12Suites.extend(aeadSuites)
 
+    #: TLS1.3 specific ciphersuites
+    tls13Suites = []
+
+    # TLS 1.3 suites are not a superset of TLS 1.2 suites, but they
+    # use the same mechanism (AEAD), so we need to remove TLS 1.3 items
+    # from the TLS 1.2 list
+    tls13Suites.append(TLS_AES_256_GCM_SHA384)
+    tls12Suites.remove(TLS_AES_256_GCM_SHA384)
+    tls13Suites.append(TLS_AES_128_GCM_SHA256)
+    tls12Suites.remove(TLS_AES_128_GCM_SHA256)
+    tls13Suites.append(TLS_CHACHA20_POLY1305_SHA256)
+    tls12Suites.remove(TLS_CHACHA20_POLY1305_SHA256)
```

Comment:
The client would send TLS 1.3 suites without advertising support for TLS 1.3 (as it was set to use TLS 1.2)

not the end of the world, but not good form either.

The comment refers to a line almost just above - 908. I can expand the comment, though I don't have immediate idea what would be good addition.